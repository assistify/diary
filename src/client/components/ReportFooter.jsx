import React from 'react';
import { PropTypes } from 'prop-types';

import Button from 'react-bulma-components/lib/components/button';
import Container from 'react-bulma-components/lib/components/container';
import Content from 'react-bulma-components/lib/components/content';
import Footer from 'react-bulma-components/lib/components/footer';

export default function ReportFooter(props) {
  const { onClick } = props;
  return (
    <Footer>
      <Container>
        <Content style={{ textAlign: 'center' }}>
          <Button className="js-toggle-editor" onClick={onClick}>
              Made with ‚ù§ by the Assistify Team
          </Button>
        </Content>
      </Container>
    </Footer>
  );
}
ReportFooter.propTypes = {
  onClick: PropTypes.func.isRequired
};
