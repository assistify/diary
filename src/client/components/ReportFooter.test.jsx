import React from 'react';
import renderer from 'react-test-renderer';
import ReportFooter from './ReportFooter';

it('renders correctly', () => {
  const tree = renderer
    .create(<ReportFooter onClick={() => true} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
