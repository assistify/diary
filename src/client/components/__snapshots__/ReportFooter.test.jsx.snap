// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="footer"
  style={Object {}}
>
  <div
    className="container"
    style={Object {}}
  >
    <div
      className="content"
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      <button
        className="js-toggle-editor button"
        disabled={false}
        onClick={[Function]}
        style={Object {}}
        tabIndex={0}
      >
        Made with ‚ù§ by the Assistify Team
      </button>
    </div>
  </div>
</div>
`;
